index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+      <link rel="preconnect" href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&display=swap">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <nav class="toolbar">
+            <nav class="tbar">
+                <nav class="menu-bar">
+            <img src="John Doe.png" class="john-doe"> 
+            <img src="Icon - Menu.png" class="icon-Menu">
+            </nav>
+        </nav>
+         
+        <nav class="headline">
+            <nav> 
```

Comment:
- [ ] If you want to separate elements in a container you could use `div` tag because this is not a navbar. Please 🙏🏽 fix it for good practice of the HTML&CSS.